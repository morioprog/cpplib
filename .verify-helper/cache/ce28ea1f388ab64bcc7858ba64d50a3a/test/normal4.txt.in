23 1
1 1
1 3
1 8
1 9
1 6
1 6
1 8
1 4
1 2
1 2
1 7
2
2
2
2
2
2
2
2
2
2
2
2
