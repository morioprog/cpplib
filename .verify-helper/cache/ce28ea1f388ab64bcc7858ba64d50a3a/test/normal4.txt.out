1
2
2
3
4
6
6
7
8
8
9
-1
